cluster/aws/util.sh

```diff
@@ -36,7 +36,9 @@ source "${KUBE_ROOT}/cluster/aws/${KUBE_CONFIG_FILE-"config-default.sh"}"
 source "${KUBE_ROOT}/cluster/common.sh"
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-ALLOCATE_NODE_CIDRS=true
+if [[ "${NETWORK_PROVIDER:-}" != "cni" ]]; then
```

Comment:
Some CNI plugins _will_ use the k8s API though.  Doesn't flannel do this today?  My kopeio-routing definitely does (though I'm still leveraging kubenet!)